src/asciidocParser.ts

```diff
@@ -134,6 +134,23 @@ export class AsciidocParser {
     }
   }
 
+  public static load (textDocument: SkinnyTextDocument): Asciidoctor.Document {
+    const memoryLogger = processor.MemoryLogger.create()
+    processor.LoggerManager.setLogger(memoryLogger)
+    asciidoctorFindIncludeProcessor.resetIncludes()
+    const baseDir = AsciidocTextDocument.fromTextDocument(textDocument).getBaseDir()
+    return processor.load(textDocument.getText(), {
+      attributes: {
+        'env-vscode': '',
+        env: 'vscode',
+      },
+      sourcemap: true,
+      safe: 'unsafe',
+      parse: true,
+      ...(baseDir && { base_dir: baseDir }),
+    })
```

Comment:
It might be worth to declare a function that build the "default" Asciidoctor options:


```js
const options = getDefaultAsciidoctorOptions(baseDir)
```